conf/routes

```diff
@@ -83,5 +83,7 @@ GET         /Voucher                         controllers.Homepage.index
 GET         /offers                          controllers.Offers.offers
 GET         /:edition/offers                 controllers.Offers.offersPage(edition: String)
 
+GET         /offersUK                        controllers.Offers.offersTestUKPage
```

Comment:
so if this is just a temporary test, worth to add a comment to the routes file just to say it will be removed in future